RELEASE.md

```diff
@@ -1,87 +1,12 @@
-# *CDM Model - RoundToPrecisionRemoveTrailingZeros Function*
```

Comment:
This has not been released yet, so can you add the to the release note rather than overwrite

Reply:
done